app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php

```diff
@@ -1260,40 +1278,31 @@ public function testGetMediaGalleryImagesMerging()
 
     public function testGetCustomAttributes()
     {
-        $interfaceAttributeCode = 'price';
-        $customAttributeCode = 'color';
-
-        $this->resource
-            ->method('getCustomAttributesCodes')
-            ->willReturn([$customAttributeCode]);
-        $this->model->setData($interfaceAttributeCode, 10);
+        $priceCode = 'price';
+        $colorAttributeCode = 'color';
```

Comment:
same as with "name" and "description" above. you probably reverted both to the previous version